Sources/SwiftCurrent_SwiftUI/Models/WorkflowViewModel.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  WorkflowViewModel.swift
+//  SwiftCurrent_SwiftUI
+//
+//  Created by Megan Wiemer on 7/13/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+
+import SwiftCurrent
+import SwiftUI
+
+@available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
+final class WorkflowViewModel: ObservableObject {
+    @Published var body = AnyView(EmptyView())
+    var isPresented: Binding<Bool>?
```

Comment:
I think `isPresented` is not a very accurate description of what this binding does. Case in point if you want to relaunch your workflow you toggle it off, then on. 

I don't necessarily have a great suggestion for a replacement but I'll humbly suggest `isLaunched` in place of `isPresented`

Reply:
Here are some iterations:
```swift
WorkflowView(isPresented: .constant(true)) // Current
WorkflowView(isLaunched: .constant(true)) // Suggested by Tyler
WorkflowView(launch: .constant(true))
WorkflowView(run: .constant(true))
WorkflowView(activate: .constant(true))
```

Reply:
I think i'm leaning towards `isLaunched`